docker/gn-postgres/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ volumes:
 
 services:
   geonetwork:
-    image: geonetwork:3.99.0
+    image: geonetwork:latest
```

Comment:
Unless I'm wrong. A difference with the docker compose for CAS / LDAP, is that CAS / LDAP use the local build, while this other docker compose uses the latest official image in Docker Hub, no?

Should not be used also the local build?

Reply:
These seem to be test environments; not sure why they were written this way. You are correct that these use the official docker image.